app/src/main/java/com/example/taskyapplication/agenda/presentation/components/AgendaDeleteComponents.kt

```diff
@@ -0,0 +1,153 @@
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.example.taskyapplication.R
+import com.example.taskyapplication.ui.theme.TaskyDesignSystem.Companion.taskyColors
+import com.example.taskyapplication.ui.theme.TaskyTypography
+
+@Composable
+fun AgendaItemDeleteTextButton(
+    modifier: Modifier = Modifier,
+    onClick: () -> Unit,
+    itemToDelete: String
+) {
+    Column(
+        modifier = modifier
+            .fillMaxWidth()
+    ) {
+        HorizontalDivider(
+            modifier = Modifier
+                .padding(top = 8.dp),
+            color = taskyColors.surfaceContainerHigh,
+            thickness = 1.dp,
+        )
+        TextButton(
+            onClick = onClick,
+            modifier = Modifier
+                .fillMaxWidth()
+                .align(Alignment.CenterHorizontally)
+        ) {
+            Text(
+                text = stringResource(R.string.delete_text_button, itemToDelete).uppercase(),
+                style = TaskyTypography.labelSmall,
+                color = taskyColors.error
+            )
+        }
+    }
+}
+
+// delete bottom sheet
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun DeleteItemBottomSheet(
+    modifier: Modifier = Modifier,
+    onDeleteTask: () -> Unit = {},
+    onCancelDelete: () -> Unit = {}
+) {
+    ModalBottomSheet(
+        modifier = modifier,
+        onDismissRequest = { onCancelDelete() },
+        sheetState = rememberModalBottomSheetState(),
+        containerColor = taskyColors.surface,
+        content = {
+            Column(
+                modifier = Modifier
+                    .background(color = taskyColors.surface)
+                    .fillMaxWidth()
+                    .padding(horizontal = 16.dp)
+                    .padding(vertical = 24.dp),
+                horizontalAlignment = Alignment.CenterHorizontally
+
+            ) {
+                Text(
+                    text = stringResource(R.string.delete_bottom_sheet_heading),
+                    style = TaskyTypography.headlineMedium,
+                    color = taskyColors.primary
+                )
+                Text(
+                    modifier = Modifier.padding(top = 8.dp),
+                    text = stringResource(R.string.delete_bottom_sheet_sub_heading),
+                    style = TaskyTypography.bodyMedium,
+                    color = taskyColors.onSurface
+                )
+                Row(
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .padding(vertical = 16.dp),
+                    verticalAlignment = Alignment.CenterVertically,
+                    horizontalArrangement = Arrangement.SpaceEvenly
+                ) {
+                    OutlinedButton(
+                        modifier = Modifier
+                            .height(52.dp)
+                            .width(156.dp),
```

Comment:
Does this button have exactly those dimensions on all devices? Looking at the mockups I wouldn't say so, it should simply be about 50% of the width of the bottom sheet (+ spacing). Such hardcoded units can really mess up responsiveness. Especially if you're in a Row like here, it typically makes more sense to work with `Modifier.weight(1f)` for both buttons to give them equal width.

For the height I'd just leave it at how much space it needs (so not giving a height value at all here). Then the text size and button padding will dictate the height. But right now increasing the text size can cut off certain parts since the height is hardcoded.